backend/app-orchestrator-service/api/src/main/kotlin/IntegratedApplications.kt

```diff
@@ -115,7 +115,10 @@ abstract class IntegratedApplicationApi<ConfigType>(
 }
 
 @Serializable
-class IAppsRetrieveConfigRequest<ConfigType>(val providerId: String)
+class IAppsRetrieveConfigRequest<ConfigType>(
+    val provider: String,
+    val product: String,
```

Comment:
I think `product` is a bad name in this case. Typically a product is a combination of (name, category, provider). It is not obvious what "product" is in this case.

Reply:
Renamed to `productId` since that seems to be the most used (even though it is actually the product name).